docs/index.md

```diff
@@ -0,0 +1,48 @@
+# strawberry-django-plus
+
```

Comment:
IDK why but some labels wont show up.
Maybe it is will work only in the parent repo.

Reply:
works now